src/libvideo.debug/libvideo.debug.csproj

```diff
@@ -61,6 +53,10 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\libvideo.compat\libvideo.compat.csproj">
```

Comment:
Sorry, could you please revert the changes to this file? I remember referencing `YoutubeExtractor` instead of the compatibility API on purpose so I could flesh out any bugs in this lib vs. YoutubeExtractor.